tests/keras/backend/backend_test.py

```diff
@@ -1655,6 +1655,34 @@ def test_bias_add(self):
         with pytest.raises(ValueError):
             K.bias_add(x, b, data_format='channels_middle')
 
+    @pytest.mark.skipif(K.backend() == 'theano',
+                        reason='Theano behaves differently '
+                               'because of the broadcast.')
+    @pytest.mark.parametrize('axis', [1, -1])
+    @pytest.mark.parametrize('x_shape', [(3, 2, 4, 5), (3, 2, 4)])
+    def test_batch_normalization(self, axis, x_shape):
+        other_shape = [1] * len(x_shape)
+        other_shape[axis] = x_shape[axis]
+        other_shape = tuple(other_shape)
+        x_np = np.random.random(x_shape)
+        mean_np = np.random.random(other_shape)
```

Comment:
How about the following?
```
mean, var, beta, gamma = [np.random.random(other_shape).astype(np.float32)
                          for _ in range(4)]
```

Reply:
I suppose it's a style matter, I'd prefer avoid breaking lines unless necessary. I actually wanted to use #11265 for this test of batchnorm.